Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM php:8.0-cli as rector
-WORKDIR /rector
+ARG PHP_VERSION=8.0
+FROM php:${PHP_VERSION}-cli as base
+
+RUN apt-get update && apt-get install -y \
+    libzip4 \
+    libicu63 \
```

Comment:
I had the impression we need those as runtime requirements; it that really the case?

Reply:
Honestly, im not sure, they must be there for reason 😄